spec/services/analytics_spec.rb

```diff
@@ -311,7 +311,7 @@
       let(:expected_attributes) do
         {
           sp_request: {
-            component_values: { 'http://idmanagement.gov/ns/assurance/ial/1' => true },
+            component_values: { 'L1' => true },
```

Comment:
IMO keeping "ial/1" makes it easier to identify, esp compared to "aal/2", etc? open to feedback here

Reply:
My only hesitation is labeling something as `ial/2` when it doesn't conform to IAL2

Reply:
Yeah, but that's a pre-existing issue with the thing being logged. I don't think we should worry too much if all we're doing is chomping off a known prefix to save a little noise in the logs.

Reply:
OK, switched to just clipping the `http://idmanagement.gov/ns/assurance/` off the front in a1729f20b.